packages/big-design/src/components/Dropdown/Dropdown.tsx

```diff
@@ -145,7 +144,7 @@ export const Dropdown = memo(
       (group: DropdownItemGroup) => {
         return (
           <>
-            <ListGroupHeader>{group.label}</ListGroupHeader>
+            <StyledListGroupHeader role="group">{group.label.toUpperCase()}</StyledListGroupHeader>
```

Comment:
I added it for the tests as they were failing without it (but they can easily be updated), and because the `ListGroupHeader` element has that attribute added while the `StyledListGroupHeader` does not.

Reply:
Per video conversation, updating tests and removing this role, also adding `aria-hidden` to the `ListGroupHeader` component